contents/tutorials/build-site-app.md

```diff
@@ -0,0 +1,191 @@
+---
+title: How to build a site app
```

Comment:
Will defer to you and @andyvan-ph on this, but it may be worth tweaking the title to include that this app is a feedback app. How to build a site app is a good title as is, but we know feedback apps are something people will want.

Reply:
As mentioned [here](https://posthog.slack.com/archives/C01V9AT7DK4/p1666891672762539?thread_ts=1666735662.272539&cid=C01V9AT7DK4), I will plan to write another tutorial about getting more feedback (through apps) in PostHog that I think better covers that demand.

Reply:
Ah, perfect <3